src/frontends/paddle/tests/test_models/gen_scripts/generate_place_test_model.py

```diff
@@ -34,11 +35,43 @@ def paddle_rnn_lstm(input_size, hidden_size, layers, direction):
             feed={'x': np.ones([4, 3, input_size]).astype(np.float32)},
             fetch_list=[y, h, c],
             program=main_program)
-        saveModel("place_test_model", exe, feedkeys=[data],
-                  fetchlist=[y, h, c, relu_1, relu_2, relu_3],
-                  inputs=[np.ones([4, 3, input_size]).astype(np.float32)],
-                  outputs=[outs[0], outs[1], outs[2]], target_dir=sys.argv[1],
-                  use_static_api=True)
+
+        if paddle.__version__ >= '2.0.0':
+            feed_vars = [data]
+            fetch_vars = [y, h, c, relu_1, relu_2, relu_3]
+            saveModel("place_test_model", exe, feedkeys=feed_vars,
+                    fetchlist=fetch_vars,
+                    inputs=[np.ones([4, 3, input_size]).astype(np.float32)],
+                    outputs=[outs[0], outs[1], outs[2]], target_dir=sys.argv[1],
+                    use_static_api=True)
+            path_prefix = os.path.join(sys.argv[1], "place_test_model", "place_test_model")
+            program, feed_target_names, fetch_targets = paddle.static.io.load_inference_model(path_prefix, exe)
+
+            from paddle.fluid import core
+            condition = lambda v : not v.persistable and v.name != "transpose_1.tmp_1" and v.name != "transpose_0.tmp_1"
+            vars_ = list(filter(condition, program.list_vars()))
+            vars_name = [v.name for v in vars_]
+            vars_name_file = os.path.join(sys.argv[1], "place_test_model", "vars_name.txt")
+            print(">>>>>>>>", vars_name_file)
+            # print the absolute path of vars_name_file
+            print(">>>>>>>>", os.path.abspath(vars_name_file))
+            with open(vars_name_file, 'w') as f:
+                for name in vars_name:
+                    f.writelines(f"{name}\n")
+
```

Comment:
Can you add test code here to read and print the content of 'vars_name_file' to make sure it is generated successfully?

Reply:
added

Reply:
It looks like it was generated successfully.

<img width="1868" alt="image" src="https://github.com/openvinotoolkit/openvino/assets/55493212/6f9a0903-2d33-441f-98b8-6cde7930a8ab">